src/app/creator-profile-page/creator-diamonds/creator-diamonds.component.ts

```diff
@@ -35,12 +51,42 @@ export class CreatorDiamondsComponent implements OnInit {
         this.isLoading = false;
       });
   }
-
   counter(num: number) {
     return Array(num);
   }
 
-  usernameTruncationLength(): number {
-    return this.globalVars.isMobile() ? 10 : 12;
+  getNoDiamondsMessage() {
+    return this.showDiamondsGiven
+      ? `@${this.profile.Username} has not given any diamonds yet.`
+      : `No has sent @${this.profile.Username} a diamond yet.`;
```

Comment:
`@${this.profile.Username} has not received any diamonds yet.`

Reply:
:facepalm: thanks!